planning/mv-params.json

```diff
@@ -0,0 +1,151 @@
+the user submits something that has this format:
+
+{
+    "includes": [
+	{
+	    "name": "common",
+	    "params": [
+		{ "arg": "foo", "vals": [ "val1", "val2" ], "role": "all" },
+		{ "arg": "bar", "vals": [ "val3", "val4" ], "role": "client" }
+	    ]
+	},
+	{
+	    "name": "scenario1",
+	    "params": [
+		{ "arg": "zorg", "vals": [ "val5", "val6" ], "role": "server" },
+		{ "arg": "alpha", "vals": [ "val7", "val8" ], "role": "infra" }
+	    ]
+	},
+	{
+	    "name": "scenario2",
+	    "params": [
+		{ "arg": "beta", "vals": [ "val9", "val10" ], "role": "all" }
+	    ]
+	}
+    ],
+    "sets": [
+	[
+	    { "include": "common" },
```

Comment:
"common" becomes "include" here (note the singular vs. plural forms of include)

Reply:
yes!